extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -374,10 +374,10 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       const clonedNode = this.cloneLightboxableElement_(element);
       const descText = this.manager_.getDescription(element);
       const metadata = {
-        descriptionText: descText,
-        tagName: clonedNode.tagName,
+        descriptionText: /** @type {string} */ (descText),
```

Comment:
return type of `getDescription` should handle this. casting doesn't seem right here.